examples/neighbors/approximate_nearest_neighbors.py

```diff
@@ -207,18 +211,25 @@ def test_transformers():
     assert_array_almost_equal(Xt0.toarray(), Xt2.toarray(), decimal=5)
 
 
-def load_mnist(n_samples):
+def load_mnist(n_samples_list):
     """Load MNIST, shuffle the data, and return only n_samples."""
-    mnist = fetch_openml(data_id=41063)
```

Comment:
No no, we just need to pass the version argument to `fetch_openml`. `version=1` could be fine.

Reply:
I got it.
But I also tried `fetch_openml(data_id=41063, version=1)` and failed due to an error (ValueError: ... you can only specify a numeric data_id or a version, not both).

Reply:
OK, you are right. So basically if we are sure that this version of MNIST is fine, we can use data_id and filter out the UserWarning manually.

Do we know what's the difference between this version of MNIST dataset and the following? Maybe searching on OpenML github would give some insights. If that version is fine it might be an issue with OpenML as well.

Reply:
Why not pass the dataset name + version id instead?

Reply:
> Why not pass the dataset name + version id instead?

The problem seems to be that particular name + version id were disabled by OpenML (and therefore raise the warning). Whether there it was done for a valid reason or not I'm not sure.